qt/settingsdialog.py

```diff
@@ -1947,6 +1947,14 @@ def btnSnapshotsPathClicked(self):
 
                 self.config.removeProfileKey('snapshots.path.uuid')
 
+            if tools.filesystem(path).startswith('ntfs'):
+                text = _('Warning: {0} is formatted as NTFS which has known incompatabilities '
+                         'with unix style file systems.\n\n'
+                         'Are you sure you want to use it as your backup destination?')
+                question = text.format(path)
```

Comment:
Imagine our translators using the web-frontend at translators.codeberg.org. Some of them have no coding background. They also need more context. Because of that it is important to give placeholders good names. The details are [hidden in our docs](https://github.com/bit-team/backintime/blob/dev/common/doc-dev/2_localization.md#instructions-for-the-translation-process).

```suggestion
                text = _('Warning: {path} is formatted as NTFS which has known incompatibilities '
                         'with Unix style file systems.'
                         'Are you sure you want to use it as your backup destination?')
                question = text.format(path=path)
```

I also removed the two `\n` because of our translators. I am not sure if, so please checkout if the dialog box does wrap the text by itself. And while you are checking it please create a screenshot of message box and attach it to the PR. If the text is not wrapped, please simply make two translatable strings like this:

    text = _('foo') + '\n\n' + _('bar')

Beside the text, the translators will also get a screenshot of the message box. So they will understand.

It is damn hard to get translators and keep them motivated. So I try to make it as easy as possible for them. 😄